src/cli/mod.rs

```diff
@@ -69,7 +70,8 @@ enum Command {
 
 #[derive(Clap)]
 struct NodeOpts {
-    address: String,
+    /// Comma separated list of IP:PORT pairs. Overrides default initial nodes.
+    addresses: Option<String>,
```

Comment:
I think you can use `Vec<String>` here, rather than having to split a string yourself. See https://github.com/clap-rs/clap_derive/blob/master/examples/example.rs#L43 (without the `Option<>`)

Reply:
Ah you'll want the `Option<>` anyway, perfect example then 👍

Reply:
Unfortunately this seems to be meant for something else: https://docs.rs/structopt-derive/0.3.2/src/structopt_derive/attrs.rs.html#541

Reply:
Yeah I think a flag makes sense. I don't think it's useful enough to warrant a positional arg.